src/AuthorizationPanelAmfOverlay.js

```diff
@@ -165,9 +165,12 @@ export const AuthorizationPanelAmfOverlay = (base) => class extends AmfHelperMix
     const secPrefix = this.ns.aml.vocabularies.security;
     let hasNull = false;
     for (let i = 0, len = secured.length; i < len; i++) {
-      const item = secured[i];
-      if (!this._hasType(item, secPrefix.ParametrizedSecurityScheme) &&
-        !this._hasType(item, secPrefix.SecurityScheme)) {
+      // TODO temporarily retrieve first item of security:schemes due to AMF 4 model change
+      const item = this._hasType(secured[i], secPrefix.securityRequirement)
```

Comment:
This notation severely reduce code readability. Please, split it into several lines. This is a public repo :)